internal/cli/cmd/connectivity.go

```diff
@@ -128,6 +128,7 @@ func newCmdConnectivityTest() *cobra.Command {
 	cmd.Flags().BoolVarP(&params.Debug, "debug", "d", false, "Show debug messages")
 	cmd.Flags().BoolVarP(&params.Timestamp, "timestamp", "t", false, "Show timestamp in messages")
 	cmd.Flags().BoolVarP(&params.PauseOnFail, "pause-on-fail", "p", false, "Pause execution on test failure")
+	cmd.Flags().StringVar(&params.DomainName, "domain-name", "one.one.one.one", "Domain name to use with external connectivity tests")
```

Comment:
naming suggestion: "external-target"

Reply:
Agree, let's change this to "external-target".

Reply:
done